app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php

```diff
@@ -30,6 +30,8 @@
  * @category    Mage
  * @package     Mage_CurrencySymbol
  * @author      Magento Core Team <core@magentocommerce.com>
+ *
+ * @method $this resetValues()
```

Comment:
what is it used for? what does it mean?

Reply:
Seems to be wrong commited .... can't find this in my patches. I'll check ....

Reply:
It is "used" in `Mage_CurrencySymbol_Adminhtml_System_CurrencysymbolController::resetAction()`, but this method does not exist ... not sure why, but i guess it was just to supress warnings.

Reply:
ok, we have bigger issue here. adding this doc comment doesn't hurt